### ISSUE TITLE
V1.2.0: Cant reach keycloak on authentication

### ISSUE THREAD

**Issue by TamimiGitHub** [microcks/microcks#354]
## Problem

I am just [getting started](https://microcks.io/documentation/getting-started/) with microcks v1.2.0 for evaluation. After cloning and starting up the docker image 
```
cd microcks/install/docker-compose
docker-compose up -d
```
I get the following error when I attempt to navigate to localhost:8080
<img width="1543" alt="Screen Shot 2021-02-25 at 6 27 06 PM" src="https://user-images.githubusercontent.com/6826053/109233224-7fa1a880-7797-11eb-99a9-718477e8de34.png">

This is my docker-compose file

```
version: '2'

services:
  mongo:
    image: mongo:3.4.23
    container_name: microcks-db
    volumes:
      - "~/tmp/microcks-data:/data/db"

  keycloak:
    image: jboss/keycloak:10.0.1
    container_name: microcks-sso
    ports:
      - "18080:8080"
    environment:
      KEYCLOAK_USER: "admin"
      KEYCLOAK_PASSWORD: "admin"
      KEYCLOAK_IMPORT: "/tmp/microcks-realm.json"
      KEYCLOAK_FRONTEND_URL: "http://localhost:18080/auth"
    volumes: 
      - "./keycloak-realm/microcks-realm-sample.json:/tmp/microcks-realm.json"

  postman:
    image: quay.io/microcks/microcks-postman-runtime:latest
    container_name: microcks-postman-runtime

  app:
    depends_on:
      - mongo
      - keycloak
      - postman
    image: quay.io/microcks/microcks:latest
    container_name: microcks
    ports:
      - "8080:8080"
    environment:
      - SPRING_PROFILES_ACTIVE=prod
      - SPRING_DATA_MONGODB_URI=mongodb://mongo:27017
      - SPRING_DATA_MONGODB_DATABASE=microcks
      - POSTMAN_RUNNER_URL=http://postman:3000
      - TEST_CALLBACK_URL=http://microcks:8080
      - SERVICES_UPDATE_INTERVAL=0 0 0/2 * * *
      - KEYCLOAK_URL=http://keycloak:8080/auth
      - KEYCLOAK_PUBLIC_URL=http://localhost:18080/auth

```

Not sure if there are any particular ports that needs to be exposed? 

## Environment
MacOS Catalina

Any directions or idea on what the problem might be? Thanks!

**Comment by lbroudoux** [comment]
Hi @TamimiGitHub,

Can you check that the Keycloak container is actually running?
When looking at running containers, you should have something like this:

```sh
$ docker ps
CONTAINER ID        IMAGE                                              COMMAND                  CREATED             STATUS              PORTS                                        NAMES
28107409f526        quay.io/microcks/microcks:latest                   "/deployments/run-ja…"   8 seconds ago       Up 7 seconds        8778/tcp, 0.0.0.0:8080->8080/tcp, 9779/tcp   microcks
c8eb85c85478        quay.io/microcks/microcks-postman-runtime:latest   "docker-entrypoint.s…"   32 hours ago        Up 8 seconds        3000/tcp                                     microcks-postman-runtime
81ee575a3517        mongo:3.4.23                                       "docker-entrypoint.s…"   32 hours ago        Up 8 seconds        27017/tcp                                    microcks-db
bf193b5b3c39        jboss/keycloak:10.0.1                              "/opt/jboss/tools/do…"   32 hours ago        Up 8 seconds        8443/tcp, 0.0.0.0:18080->8080/tcp            microcks-sso
```

with the logs of Keycloak being something like:
```sh
$ docker logs bf193b5b3c39
[...]
16:54:27,039 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0212: Resuming server
16:54:27,042 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0060: Http management interface listening on http://127.0.0.1:9990/management
16:54:27,043 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0051: Admin console listening on http://127.0.0.1:9990
16:54:27,043 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: Keycloak 10.0.1 (WildFly Core 11.1.1.Final) started in 34001ms - Started 689 of 994 services (708 services are lazy, passive or on-demand)
```

**Comment by TamimiGitHub** [comment]
Hey @lbroudoux thanks for pointing out this info. I checked my docker images and it looks like keycloak image was not downloaded properly (not sure how that went through since the app depends on keycloak being successfully up and running)
```
app:
    depends_on:
      - mongo
      - keycloak
      - postman
```
I deleted the image, reran my docker-compose up and everything is up and running. So we can go ahead and close this issue, thanks!
